app/models/team_performance.rb

```diff
@@ -0,0 +1,72 @@
+class TeamPerformance
+# Memo: calculates Team's Performance Score for Supervisor's Dashboard
+
+  def initialize(team)
+    @team = team
+  end
+
+  def score
+    @score = 0
+
+    if @team.comments.any?
+      comments_score
+    elsif Time.now-2.days > Season.current.starts_at && Time.now < Season.current.ends_at
```

Comment:
This is dificult to read and could do with a simplification. It's intention is to show the team as green for the first two days of the SoC?

There is the exact same line in line 22. Is it really depending on whether there are activities or comments, respectively? If not, maybe pull it out into a `early_bird_score` method?